composer.json

```diff
@@ -27,18 +27,18 @@
         "kartik-v/strength-meter": "~1.1.0",
         "matthiasmullie/minify": "~1.3.0",
         "monolog/monolog": "~1.24",
-        "php": "^7.1",
+        "php": "^7.1|^8.0",
```

Comment:
(tbd Should this already be 7.4 for example?)

Reply:
Bumped it to 7.4 and updated the minimum versions of dependencies accordingly.